### ISSUE TITLE
BUG: Evaluate model using COCO dataset 

### ISSUE THREAD

**Issue by adrianosantospb** [tristandb/EfficientDet-PyTorch#5]
Thank you for share your repository and implementation.
I'm training a model using my own dataset. The training process seems to work well, but when the model calls the evaluation function, I get this error:

`File "/home/workstation/anaconda3/envs/efficientdet/lib/python3.7/site-packages/torch/nn/parallel/scatter_gather.py", line 55, in gather_map
    return Gather.apply(target_device, dim, *outputs)
  File "/home/workstation/anaconda3/envs/efficientdet/lib/python3.7/site-packages/torch/nn/parallel/_functions.py", line 54, in forward
    assert all(map(lambda i: i.is_cuda, inputs))`

This error occurs in this call:

`             scores, labels, boxes = model(data['img'].permute(2, 0, 1).cuda().float().unsqueeze(dim=0))
`
 Some idea?
